drivers/SmartThings/zigbee-switch/src/init.lua

```diff
@@ -85,6 +93,50 @@ local device_init = function(self, device)
   end
 end
 
+local function is_mcd_device(device)
+  for _, fingerprint in ipairs(MCD_fingerprint) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
```

Comment:
we shouldn't check that the device is specific model, we should just check if the profile on the device object has multiple components